concent_api/core/views.py

```diff
@@ -82,19 +88,24 @@ def send(_request, client_message):
         force_report_computed_task = message.Message.deserialize(
             force_report_computed_task_from_database.last().data.tobytes(),
             None,
-            check_time = False
```

Comment:
Actually, now that I think about it this line is wrong.

You're removing a line that ignores the timestamp on a message deserialized from the database. We **do** want to ignore the timestamp here.

What you should be doing instead is to remove `check_time` in `api_view` decorator which decoded the outer message for the view.